mods/page_save.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+
+/**
+ * page_save module
+ *
+ * Sends the wikitext of a page to the MW API for saving
```

Comment:
> Also, it'll keep the file sizes at bay, which makes it easier to manage. 

Agreed. Although, we could also sub-divide by implementing one API module with several code modules.

With the current split I fear that it might become a bit hard to figure out which module handles what. For example, will wikitext retrieval live in the parsoid module, or the page_save module? The two modules will also depend on each other heavily.